meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -20,7 +20,6 @@ do_install_append() {
     if ${@bb.utils.contains('DISTRO_FEATURES', 'pvcamera', 'true', 'false', d)}; then
         cat ${WORKDIR}/domu-pvcamera.cfg >> ${CFG_FILE}
         # Update GUEST_DEPENDENCIES by adding camerabe after sndbe
```

Comment:
With that removing the comment above is stale. Everything else lgtm

Reply:
Done